tket2-py/README.md

```diff
@@ -1,4 +1,31 @@
-## tket2-py
+# tket2
+
+[![pypi][]](https://pypi.org/project/tket2/)
+[![codecov][]](https://codecov.io/gh/CQCL/tket2)
+[![py-version][]](https://pypi.org/project/tket2/)
+
+  [codecov]: https://img.shields.io/codecov/c/gh/CQCL/tket2?logo=codecov
+  [py-version]: https://img.shields.io/pypi/pyversions/tket2
+  [pypi]: https://img.shields.io/pypi/v/tket2
+
+TKET2 is an open source quantum compiler developed by Quantinuum. Central to
+TKET2's design is its hardware agnosticism which allows researchers and
+quantum software developers to take advantage of its state of the art
```

Comment:
I don't like "state of the art" -- what about "powerful" instead?

Reply:
:+1:
I'll change it on the other READMEs too